### DISCUSSION TITLE
Mouse cursor stops momentarily when copying a large data like image

### DISCUSSION THREAD

**Discussion by HubKing** [oae/gnome-shell-pano#166] [Q&A]
To test this, try to click a "copy" menu or button with a large image and after clicking immediately try to move the mouse pointer. If Pano is not enabled, mouse cursor moves, but if Pano is enabled, mouse cursor stops moving for a brief moment.

I guess the copying process is blocking Gnome Shell? Is this an inevitable consequence due to Gnome's extension architecture?

**Comment by oae** [comment]
This is a known issue. See https://github.com/oae/gnome-shell-pano/issues/79

I will try to optimize pano after few other features are completed
